main/design_documents/__ARM_FEATURE_MOPS.rst

```diff
@@ -0,0 +1,75 @@
+=============================================================
+Introducing extensions for memcpy family of memory operations
+=============================================================
+
+Following the announcement of the `2021 Architecture Extensions
+<https://community.arm.com/developer/ip-products/processors/b/processors-ip-blog/posts/arm-a-profile-architecture-developments-2021>`_,
+we are proposing these aditions to the ACLE to support the new instructions
+that standardise the memcpy family of operations.
```

Comment:
Please mention `Armv8.8-A and Armv9.3-A` somewhere in this paragraph to make it clear which are the version of the architecture being mentioned here.